--- /home/jun/research/dlapr/all_mutants/mockito-1f/mutants/1805/org/mockito/internal/runners/util/TestMethodsFinder.java	2022-10-22 16:02:29.717454508 -0500
+++ /home/jun/research/dlapr/all_mutants/mockito-1f/src/org/mockito/internal/runners/util/TestMethodsFinder.java	2022-10-22 16:02:47.629565375 -0500
@@ -1,21 +1,21 @@
 /*
  * Copyright (c) 2007 Mockito contributors
  * This program is made available under the terms of the MIT License.
  */
 package org.mockito.internal.runners.util;
 
 import org.junit.Test;
 
 import java.lang.reflect.Method;
 
 public class TestMethodsFinder {
     public boolean hasTestMethods(Class<?> klass) {
         Method[] methods = klass.getMethods();
         for(Method m:methods) {
-            if (true) {
+            if (m.isAnnotationPresent(Test.class)) {
                 return true;
             }
         }
         return false;
     }
 }
