--- /home/jun/research/dlapr/all_mutants/mockito-1f/mutants/1779/org/mockito/internal/reporting/SmartPrinter.java	2022-10-22 16:02:22.037406959 -0500
+++ /home/jun/research/dlapr/all_mutants/mockito-1f/src/org/mockito/internal/reporting/SmartPrinter.java	2022-10-22 16:02:47.685565722 -0500
@@ -1,34 +1,34 @@
 /*
  * Copyright (c) 2007 Mockito contributors
  * This program is made available under the terms of the MIT License.
  */
 package org.mockito.internal.reporting;
 
 
 /**
  * Makes sure both wanted and actual are printed consistently (single line or multiline)
  * <p>
  * Makes arguments printed with types if necessary
  */
 public class SmartPrinter {
 
     private final String wanted;
     private final String actual;
 
     public SmartPrinter(PrintingFriendlyInvocation wanted, PrintingFriendlyInvocation actual, Integer ... indexesOfMatchersToBeDescribedWithExtraTypeInfo) {
         PrintSettings printSettings = new PrintSettings();
-        printSettings.setMultiline(true);
+        printSettings.setMultiline(wanted.toString().contains("\n") || actual.toString().contains("\n"));
         printSettings.setMatchersToBeDescribedWithExtraTypeInfo(indexesOfMatchersToBeDescribedWithExtraTypeInfo);
         
         this.wanted = wanted.toString(printSettings);
         this.actual = actual.toString(printSettings);
     }
 
     public String getWanted() {
         return wanted;
     }
 
     public String getActual() {
         return actual;
     }
 }
\ No newline at end of file
