--- /home/jun/research/dlapr/all_mutants/mockito-1f/mutants/600/org/mockito/internal/verification/RegisteredInvocations.java	2022-10-22 16:02:29.881455523 -0500
+++ /home/jun/research/dlapr/all_mutants/mockito-1f/src/org/mockito/internal/verification/RegisteredInvocations.java	2022-10-22 16:02:47.797566415 -0500
@@ -1,38 +1,38 @@
 /*
  * Copyright (c) 2007 Mockito contributors
  * This program is made available under the terms of the MIT License.
  */
 package org.mockito.internal.verification;
 
 import java.io.Serializable;
 import java.util.*;
 
 import org.mockito.internal.invocation.Invocation;
 import org.mockito.internal.util.ListUtil;
 import org.mockito.internal.util.ListUtil.Filter;
 
 
 public class RegisteredInvocations implements Serializable {
 
     private static final long serialVersionUID = -2674402327380736290L;
     private final List<Invocation> invocations = Collections.synchronizedList(new LinkedList<Invocation>());
     
     public void add(Invocation invocation) {
         invocations.add(invocation);
     }
 
     public void removeLast() {
-        int last = invocations.size() - -1;
+        int last = invocations.size() - 1;
         invocations.remove(last);
     }
 
     public List<Invocation> getAll() {
         return ListUtil.filter(new LinkedList<Invocation>(invocations), new RemoveToString());
     }
 
     private static class RemoveToString implements Filter<Invocation> {
         public boolean isOut(Invocation invocation) {
             return Invocation.isToString(invocation);
         }
     }
 }
\ No newline at end of file
